.env.template

```diff
@@ -29,13 +29,15 @@ REACT_APP_NIMBUS_INSTALLATION_URL=https://nimbus.guide/intro.html
 REACT_APP_PRYSM_INSTALLATION_URL=https://docs.prylabs.network/docs/install/install-with-script
 # string
 REACT_APP_TEKU_INSTALLATION_URL=https://docs.teku.pegasys.tech/en/latest/HowTo/Get-Started/Build-From-Source/
+# string
+REACT_APP_TUTORIAL_URL=https://notes.ethereum.org/@launchpad/mainnet
```

Comment:
TBD: do we expect to have mainnet https://notes.ethereum.org/@launchpad/mainnet?

Pros: it's good to provide the comprehensive tutorials
Cons: it could be an attack vector

/cc @parithosh @MariusVanDerWijden what do you think?

Reply:
TBH I wouldn't rely on our barebone docs for mainnet - I don't think we've got a plan on how to keep them up to date. But we can still have such a doc that purely links to community guides + some FAQ questions if needed.